### PR TITLE
Improve documentation

### LINKED ISSUE #611: Support for VScode setup
**Describe the feature you'd like**
I think it might be useful to add some sort of support, for example a guide, on how to set up VScode to contribute to GQCP. At the moment it can be very difficult to try and set up every .json file and the qgcp.code-workspace. By documenting this in either a "contributing" section on the new docusaurus site or the current installation.md files.

**Describe what the current code offers in relation to what it lacks**
A step by step guide on how to set up the VScode settings would be the most helpful for new members I believe. But feel free to tell me what you guys think on the subject.

### PR DESCRIPTION
**Pull request by lelemmen**
**Short description**
This PR gathers some improvements to the documentation.

**Related issues**
- #607
- This PR closes #611.

### CODE REVIEW

**Comment by xdvriend** [comment]
@lelemmen Would it be okay if I contribute to this branch for the VS code setup section? That way I would be able to reference certain sections of the "getting started" page that has not yet been deployed onto the develop branch.

**Comment by lelemmen** [comment]
Of course, feel free to do so!

**Comment by xdvriend** [comment]
My latest commit should address issue #611.

File: site/docs/developer_documentation/getting_started.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+```json
+{
+    "configurations": [
+        {
+            "includePath": [
+                "${CMAKE_PREFIX}/envs/gqcp_dev/include",
+                "${CMAKE_PREFIX}/envs/gqcp_dev/include/eigen3",
+                "${workspaceFolder}/gqcp/include"
+            ]
```
Comment (xdvriend) [inline site/docs/developer_documentation/getting_started.md:185]:
```suggestion
            "includePath": [
                "${CMAKE_PREFIX}/envs/gqcp_dev/include",
                "${CMAKE_PREFIX}/envs/gqcp_dev/include/eigen3",
                "${CMAKE_PREFIX}/envs/gqcp_dev/share/cmake/pybind11
                "${workspaceFolder}/gqcp/include"
            ]
```

While trying to takcle an issue on gqcp I found out that I still had include errors when it came to pybind11. The line I added solved this for me. I don't know whether it was a personal issue or if it's useful to add it to the documentation though.

**Review by xdvriend — APPROVED** [review]
I've looked over the changes you made and agree that it looks better this way. Comes across less "clunky". I've added one suggestion, based on something I encountered earlier, which might be useful to look into. I'll already approve this PR, as everything looks good to me.

File: site/docs/developer_documentation/getting_started.md (hunk 1 above)
Comment (lelemmen) [reply site/docs/developer_documentation/getting_started.md:185]: @xdvriend The extra path you've mentioned contains no header filers, only contains CMake files. I'm not sure that adding this path was really the solution to the problem you had.

**Pull request merged**